contents/blog/startup-golioth.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Startup Spotlight: How Golioth builds IoT software"
+date: 2023-07-26
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in [PostHog for Startups](/startups) about the way they work and the challenges they face.
+
+[Golioth](https://golioth.io/) is an IoT developer platform that enables engineers to build IoT products and infrastructure quickly and easily. Golioth ensures that developers can securely capture and route sensor data, issue OTA firmware updates, and manage fleets of connected devices — but faces the unique challenge of creating cloud software for new, rapidly evolving hardware projects.
+
+“Hardware and software development are very different worlds,” says Golioth Founder and CEO Jonathan Beri. “Hardware? It’s slow. It’s hard. It’s incredibly complicated. And software too has its own unique challenges, even before you add IoT into the mix.”
+
+Golioth has adopted a nuanced approach to product development in order to combine these disciplines effectively — essentially dividing the way teams work based on whether it’s on the firmware & SDK side of the business, or in the cloud product.
+
+“On the firmware side and for our SDKs we have a longer release cycle because those updates get baked into our users’ operating systems,” says Golioth Product Manager Dylan Swartz. “If we ship features every every single day, our customers couldn’t keep up. But on the cloud side we’re much more agile. We run traditional two-week sprints, regularly adjust priorities and release regularly.”
+
+Regardless of the product however, there are four essential principles which Golioth applies universally, across all aspects of product and feature development…
```

Comment:
Can we change up for the format of this intro a bit? It needs a bit more pace / urgency – I'm thinking something like [The Guardian's Pass Notes](https://www.theguardian.com/lifeandstyle/2023/jul/19/weekend-warriors-exercise-cardiovascular) minus the snark, if that makes sense?

Reply:
I'll give it a further edit, but as discussed async that's not really the style I'm going for with this one - nor do I think this content is well suited to it. 

Not ruling it out for a future format switch -- it would be good to try it -- but not this time.